manifests/service.pp

```diff
@@ -31,6 +36,13 @@
       path        => $::path,
       notify      => Service[$service_name]
     }
+  } elsif ($initstyle == 'upstart') {
```

Comment:
It's called `upstart` but the script is in fact `sysvinit` or `init.d` script. That's quite confusing.